lib/improver/tests/utilities/test_load.py

```diff
@@ -150,6 +155,48 @@ def test_ordering(self):
         self.assertArrayAlmostEqual(result.coord_dims("latitude")[0], 2)
         self.assertArrayAlmostEqual(result.coord_dims("longitude")[0], 3)
 
+    def test_ordering_for_percentile_over_coordinate(self):
+        """Test that the cube has been reordered, if it is originally in an
+        undesirable order and the cube contains a "percentile_over"
+        coordinate."""
+        directory = mkdtemp()
+        filepath = os.path.join(directory, "temp.nc")
+        cube = set_up_percentile_temperature_cube()
+        percentile_points = np.array([10, 50, 90])
+        cube.transpose([3, 2, 1, 0])
+        iris.save(cube, filepath)
+        result = load_cube(filepath)
+        self.assertIsInstance(result, iris.cube.Cube)
+        self.assertArrayAlmostEqual(
+            result.coord("percentile_over_realization").points,
+            percentile_points)
+        self.assertArrayAlmostEqual(
+            result.coord("time").points, self.time_points)
+        self.assertArrayAlmostEqual(
+            result.coord("latitude").points, self.latitude_points)
+        self.assertArrayAlmostEqual(
+            result.coord("longitude").points, self.longitude_points)
```

Comment:
All these tests do is check that each coordinate has the predicted points in the right order, it does not check that the coordinate is associated with the right dimension, which is what this test is supposed to be doing.

Reply:
Removed.